test/langtools/tools/javac/diags/examples/PreviewFeatureUse.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018, 2019, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I did a bit of reshuffling here -- we had a single diagnostic test called `PreviewFeatureUse` which was only testing the `warn.preview.feature.use.plural` diagnostic. As a consequence, when I changed flexible constructor bodies to no longer be a preview feature, I started getting failures in `CheckExamples` because there was no test covering `warn.preview.feature.use` (e.g. w/o the trailing `.plural`). So I've added a test for both diagnostics (and renamed the old one to `PreviewFeatureUsePlural`).